appveyor.yml

```diff
@@ -119,8 +119,21 @@ after_test:
       .\cache-s3 --max-size=$env:CACHE_S3_MAX_SIZE --prefix=$env:APPVEYOR_PROJECT_NAME --git-branch=$env:APPVEYOR_REPO_BRANCH --suffix=windows -c -v info save stack work
     }
  - xcopy /q /s /e /r /k /i /v /h /y "%WORK_DIR%\daedalus" "%APPVEYOR_BUILD_FOLDER%\daedalus"
+ - mkdir deploy
+ - dir deploy
```

Comment:
Does something need to be copied to this new directory here for the the S3 deployment section to do anything interesting?

Reply:
yeah, its still unfinished